src/test/java/com/xceptance/neodymium/tests/TestCaseFilteringTest.java

```diff
@@ -0,0 +1,43 @@
+package com.xceptance.neodymium.tests;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+
+import org.aeonbits.owner.ConfigFactory;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.runner.JUnitCore;
+import org.junit.runner.Result;
+
+import com.xceptance.neodymium.testclasses.filtering.TestCaseFiltering;
+import com.xceptance.neodymium.util.Neodymium;
+
+public class TestCaseFilteringTest extends NeodymiumTest
+{
+    @BeforeClass
+    public static void beforeClass() throws IOException
+    {
+        final String fileLocation = "config/test-filtering-neodymium.properties";
+
+        Map<String, String> properties = new HashMap<>();
```

Comment:
Enhance readability
```
        Map<String, String> properties = new HashMap<>();
        properties.put("neodymium.testNameFilter", ".*#(shouldBeExecuted|shouldBeExecutedForDataSetWithExecutableId) :: executable.*");
```

Reply:
Done, thanks for advice

Reply:
see the comment above, adjustment needed after the other one is done :-)

Reply:
Done